### DISCUSSION TITLE
Add multi node scaling 2D

### DISCUSSION THREAD

**Discussion by rpgshooter** [godotengine/godot-proposals#8481] [Editor]
### Describe the project you are working on

A 2d game

### Describe the problem or limitation you are having in your project

Having to resize both sprite scale and collision scale separately using the point resize.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

The ability to scale both at the same time. Makes it much more easy to prototype levels.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

you click on the parent node and whatever is below that parent node can be both re-scaled. EX: a collision node and a sprite node

### If this enhancement will not be used often, can it be worked around with a few lines of script?

not really

### Is there a reason why this should be core and not an add-on in the asset library?

it would make prototyping and other aspects of 2d game development much quicker.

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.

> you click on the parent node and whatever is below that parent node can be both re-scaled. EX: a collision node and a sprite node

Collision shapes shouldn't be scaled, so this has implications if the editor were to support this:

- The editor would require special casing to resize collision shapes when using the Scale tool.
- The collision shape would have to be made unique once you resize it this way, so that it doesn't affect other collision shapes using the same resource in the scene.
